config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -47,6 +47,63 @@ presubmits:
       testgrid-dashboards: sig-testing-misc
       description: Runs conformance tests using kind and the conformance image
   - name: pull-kubernetes-conformance-kind-ipv6
+    branches:
+    - master
+    always_run: false
+    skip_report: false
+    max_concurrency: 8
+    optional: true
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      # run on the ubuntu node pool, which has ipv6 modules
+      tolerations:
+      - key: "dedicated"
+        operator: "Equal"
+        value: "ubuntu"
+        effect: "NoSchedule"
+      nodeSelector:
+        dedicated: "ubuntu"
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191218-bcc4f6d-master
+        env:
+        # enable IPV6 in bootstrap image
+        - name: "DOCKER_IN_DOCKER_IPV6_ENABLED"
+          value: "true"
+        # tell kind CI script to use ipv6
+        - name: "IP_FAMILY"
+          value: "ipv6"
+        args:
+        - "--job=$(JOB_NAME)"
```

Comment:
Nit: Is there a reason for not using [pod utilities](https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md) (`decorate: true`)? [`bootstrap.py` is deprecated](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/86389/pull-kubernetes-kubemark-e2e-gce-big/1207577736780976128/#1:build-log.txt%3A2)

Reply:
I've copy pasted, not familiar with `decorate`, can you link me an example so I can modify the jobs accordingly?

Reply:
it seems that all the e2e KIND jobs are using bootstrap.py, but the new ones, but those are using a different container image.
I think that it will be better to do the migration to podutilities of all the KIND jobs in a follow up PR, to have consistency, what do you think @BenTheElder ?

Reply:
@aojea i have some notes that may help - https://gist.github.com/dims/c1296f8ed42238baea0a5fcae45f4cf4#file-notes-md

Reply:
> I think that it will be better to do the migration to podutilities of all the KIND jobs in a follow up PR, to have consistency, what do you think @BenTheElder ?

yes, please. let's minimize the diffs that are not related.

> Nit: Is there a reason for not using pod utilities (decorate: true)? bootstrap.py is deprecated

nearly all jobs are because nobody is signing up to make this migration.

we also still have bugs blocking data collection working effectively so i'm hesitant to encourage moving.

Reply:
I've experimentally migrated a more recent job to a much more streamlined podutils config + image but I do not think we should migrate the older kubernetes release jobs @aojea, they will be phased out. we should avoid touching jobs that are not for master.